src/figuresClasses.ts

```diff
@@ -1,19 +1,95 @@
-export interface Figure {
+function Rounding(value: number): number {
```

Comment:
```suggestion
function getRoundedValue(value: number): number {
```
name of function should describe action - what it do or what it give